routes/clubs.js

```diff
@@ -885,6 +885,110 @@ router.patch("/:clubId/mrfs/:year/:month", checkAuth(function(req, res, auth){
 	// };
 });
 
+router.patch("/:clubId/mrfs/:year/:month/submit", checkAuth(function(req, res, auth){
+	getClub(req.params.clubId, {goals: 1, division_id: 1}, function(club){
+		if(club == null){
+			auth(false);
+			return;
+		}
+
+		var user = res.locals.user;
+		res.locals.club = club;
+
+		if(utils.isInt(req.params.year) && utils.isInt(req.params.month)){
+			req.params.year = Number(req.params.year);
+			req.params.month = Number(req.params.month);
+			var year = req.params.year;
+			var month = req.params.month;
+
+			if(month > 0 && month <= 12){
+				auth(club._id.equals(user.club_id) && user.access.club > 0);
+			}else{
+				auth(false);
+			}
+		}else{
+			auth(false);
+		}
```

Comment:
Add checks on user access:
Club Secretary can Submit (0 -> 1)
District Secretary can Unsubmit (1 -> 0), Confirm (1 -> 2).

Going from 2 -> 1 should be much stricter. We can't disable that feature, perhaps we can create a SOP (standard operating procedure) where the D.Sec needs the Tech Chair to manually edit it in the database, until a better solution is designed

Reply:
When you mean we can't disable that feature, do you mean it as how we intentionally designed it or just a part of the backend prevents this?